clang/test/CodeGen/ignore-overflow-pattern-false-pos.c

```diff
@@ -0,0 +1,63 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fsanitize=signed-integer-overflow,unsigned-integer-overflow -fsanitize-undefined-ignore-overflow-pattern=all %s -emit-llvm -o - | FileCheck %s
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fsanitize=signed-integer-overflow,unsigned-integer-overflow -fsanitize-undefined-ignore-overflow-pattern=all -fwrapv %s -emit-llvm -o - | FileCheck %s
+
+// Check for potential false positives from patterns that _almost_ match classic overflow-dependent or overflow-prone code patterns
+extern unsigned a, b, c;
+extern int u, v, w;
```

Comment:
are these unused?

Reply:
u, v, and we are unused. Removing them in a follow-up PR (also touching-up documentation). 

Thanks